### DISCUSSION TITLE
我想在播放界面添加对nplayer的调用，请问该怎么实现呢？谢谢！

### DISCUSSION THREAD

**Discussion by l1zheng** [spencerwooo/onedrive-vercel-index#527] [Q&A] [answered]
平板上只有nplayer，我想在播放页面把IINA替换成nplayer或者添加一个npayer按钮，请问应该修改哪部分代码呢？感谢！

**Comment by spencerwooo** [comment, marked as answer]
https://github.com/spencerwooo/onedrive-vercel-index/blob/7eb786025382146b6dbea478929c8e491e917594/components/previews/VideoPreview.tsx#L159-L163

这里是 IINA 的 URL scheme 定义，但是如果修改的话你需要知道 nPlayer 的 URL scheme。

**Reply by l1zheng** [reply]
好的，我去查一下资料，谢谢🙏
